### ISSUE TITLE
Reduce APK File Size

### ISSUE THREAD

**Issue by sureapps** [bravobit/FFmpeg-Android#155]
can we manage in one .aab verion in manage both file.
so can you help me reduced aab file size. My aab file size is 27 MB .But when i use 
   
    aaptOptions {
        ignoreAssetsPattern "!arm"
    }

 OR

    aaptOptions {
        ignoreAssetsPattern "!x86"
    }
My aab file size is 18MB so thats working fine but this solution is only one supported cpu architecture but when i live on google play store so how to manage 64-bit or 32- bit supported cpu architecture in single aab file on android

Thank you

**Comment by alexcohn** [comment]
This project is not AAB friendly. I strongly suggest that you switch to https://github.com/tanersener/mobile-ffmpeg which is also compatible with Android 10 and provides true 64-bit support.

**Comment by usmanrana07** [comment]
@alexcohn I tried mobile-ffmpeg library but if apk size was 60MB with this library then it got raised to 139MB with mobile-ffmpeg. Besides that it has only execute command method. No listener is there to observe progress,sucess,failure etc,

**Comment by sahujaunpuri** [comment]
I have also created 1 app using this library.
The size of the debug apk is 37 mb and the size of the  release apk is 71 mb.
I don't understand what to do.

**Comment by sureapps** [comment]
> No listener is there to observe progress,sucess,failure etc,

yes that is the issue we have also progress method in retrieve to fps or frame so we have set progress dialog in percentage.
so we need to listener is there to observe progress,sucess,failure etc,
